libzotero/libzotero.py

```diff
@@ -26,6 +26,33 @@
 import time
 from libzotero.zotero_item import zoteroItem as zotero_item
 
+def decode_zotero_str(raw, max_iters=3):
```

Comment:
> What operating system are you on?

I'm on Ubuntu 16.04.

> It has been observerd that Zotero encodes its strings sometimes twice recursively. The modifications of this commit try to cope with that by decoding each string up to three times.

What makes you think this? Is this a conclusion based on the fact the decoding multiple times resolves the encoding issues, at least in some cases? Or is it a known bug in the Zotero code?

I'll think about this. The combination of my odd hack (clearly dating from before understood how encoding really works) and your fix is nonsensical; but it may be that your fix on its own is correct. But I'd like to do some more tests before I push another fix that actually doesn't fix things.

(As you may have noticed, I don't have that much time to work Qnotero anymore. But I'll get to it!)